verification_token/models.py

```diff
@@ -41,7 +41,7 @@ def _create(self, obj, slug=None, extra_data=None, key_generator_kwargs=None, **
             content_type=ContentType.objects.get_for_model(obj.__class__),
             object_id=obj.pk,
             slug=slug,
-            expiration_in_minutes=expiration_in_minutes,
+            expires_at=(timezone.now() + timedelta(minutes=expiration_in_minutes)) if expiration_in_minutes else None,
```

Comment:
You must use the same timezone as project, since relative time interval is passed here. Use `localtime`.